frontend/src/css/modules/analytics.scss

```diff
@@ -0,0 +1,38 @@
+li.analytics-list {
```

Comment:
@aka-jain: We can write something like this. 
**Case 1: For the same element we can write** 
```
li {
    &.analytics-list {}
}
```
**Case 2: for parent child relationship we can write like this**
```
li {
    & .analytics-list {}
}
```
or 
```
li {
    analytics-list {}
}
```